### ISSUE TITLE
NullPointerException when runs dataflow analysis with joern.

### ISSUE THREAD

**Issue by popthink** [plume-oss/plume#171]
1. Generate 'cpg.bin' without Dataflow Pass of Plume.
2. Open 'cpg.bin' and run.ossdataflow with joern

Note.
 - In Plume, there is not any exception.

Exception staktrace:
```run.ossdataflow 
2021-04-12 14:42:25.908 WARN ParallelCpgPass: pass failed
java.util.NoSuchElementException: null
	at scala.collection.immutable.BitmapIndexedMapNode.apply(HashMap.scala:635) ~[org.scala-lang.scala-library-2.13.4.jar:?]
	at scala.collection.immutable.HashMap.apply(HashMap.scala:132) ~[org.scala-lang.scala-library-2.13.4.jar:?]
	at io.shiftleft.dataflowengineoss.passes.reachingdef.DataFlowSolver.$anonfun$calculateMopSolutionForwards$1(DataFlowSolver.scala:24) ~[io.shiftleft.dataflowengineoss_2.13-1.3.116.jar:1.3.116]
	at scala.collection.StrictOptimizedIterableOps.flatMap(StrictOptimizedIterableOps.scala:117) ~[org.scala-lang.scala-library-2.13.4.jar:?]
	at scala.collection.StrictOptimizedIterableOps.flatMap$(StrictOptimizedIterableOps.scala:104) ~[org.scala-lang.scala-library-2.13.4.jar:?]
	at scala.collection.mutable.HashSet.flatMap(HashSet.scala:31) ~[org.scala-lang.scala-library-2.13.4.jar:?]
	at io.shiftleft.dataflowengineoss.passes.reachingdef.DataFlowSolver.calculateMopSolutionForwards(DataFlowSolver.scala:22) ~[io.shiftleft.dataflowengineoss_2.13-1.3.116.jar:1.3.116]
	at io.shiftleft.dataflowengineoss.passes.reachingdef.ReachingDefPass.runOnPart(ReachingDefPass.scala:17) ~[io.shiftleft.dataflowengineoss_2.13-1.3.116.jar:1.3.116]
	at io.shiftleft.dataflowengineoss.passes.reachingdef.ReachingDefPass.runOnPart(ReachingDefPass.scala:11) ~[io.shiftleft.dataflowengineoss_2.13-1.3.116.jar:1.3.116]
	at io.shiftleft.passes.ParallelCpgPass.$anonfun$enqueueInParallel$2(ParallelCpgPass.scala:57) ~[io.shiftleft.codepropertygraph_2.13-1.3.116.jar:1.3.116]
	at io.shiftleft.passes.ParallelCpgPass.$anonfun$enqueueInParallel$2$adapted(ParallelCpgPass.scala:52) ~[io.shiftleft.codepropertygraph_2.13-1.3.116.jar:1.3.116]
	at io.shiftleft.passes.ParallelIteratorExecutor.$anonfun$map$2(ParallelIteratorExecutor.scala:11) ~[io.shiftleft.codepropertygraph_2.13-1.3.116.jar:1.3.116]
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:672) ~[org.scala-lang.scala-library-2.13.4.jar:?]
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:431) ~[org.scala-lang.scala-library-2.13.4.jar:?]
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183) ~[?:?]```

**Comment by DavidBakerEffendi** [comment]
There was a similar bug which no longer is an issue after v0.4.4, please see if this is still happening in v0.4.4.

Part of the fix in v0.4.4 was that there were some missing edges in the stubs of external classes.

**Comment by popthink** [comment]
Fixed in v0.4.4.

Close this issue.
